openshift/system/config/database.yml

```diff
@@ -9,11 +9,11 @@ base: &default
     timezone: 'UTC'
 <% else %>
   adapter: mysql2
+  collation: utf8_bin
 <% end %>
 
   url: <%= ENV['DATABASE_URL'] %>
```

Comment:
I think we should remove the `:url` parameter
Rails will auto-merge DATABASE_URL with other

In fact it has even `PRODUCTION_DATABASE_URL` `#{Rails.env.to_s.upcase}_DATABASE_URL `
https://github.com/rails/rails/blob/d72f940d1612aa8b905214c06115db48af41e8ec/activerecord/lib/active_record/database_configurations.rb#L264

We can take advantage of this later ;)
Not the purpose of this PR of course

Reply:
I tried. I doesn't work.